Singularity/Singularity/Platform/PlatformActionState.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Singularity.Platform
+{
+    /// <summary>
+    /// Each PlatformAction is in either one of the following States.
+    /// </summary>
+    public enum PlatformActionState
+    {
+        /// <summary> Available: meaning it is currently being executed, so not
```

Comment:
I think the comment description here is wrong

Reply:
In what way?

Reply:
the comment for available and active are the same